freqtrade/rpc/rpc.py

```diff
@@ -339,7 +340,10 @@ def _rpc_trade_statistics(
             self, stake_currency: str, fiat_display_currency: str,
             start_date: datetime = datetime.fromtimestamp(0)) -> Dict[str, Any]:
         """ Returns cumulative profit statistics """
-        trades = Trade.get_trades([Trade.open_date >= start_date]).order_by(Trade.id).all()
+        trade_filter = \
+            (Trade.is_open.is_(False) & (Trade.close_date >= start_date)) | \
+            Trade.is_open.is_(True)
```

Comment:
If you wrap the whole expression in an additional set of parenteses, you can "save" the `\` line concatenation - which i find rather ugly.